pkg/dataload/constants.go

```diff
@@ -18,11 +18,13 @@ package dataload
 type DataLoadPhase string
 
 const (
-	DataLoadPhaseNone     DataLoadPhase = ""
-	DataLoadPhasePending  DataLoadPhase = "Pending"
-	DataLoadPhaseLoading  DataLoadPhase = "Loading"
-	DataLoadPhaseComplete DataLoadPhase = "Complete"
-	DataLoadPhaseFailed   DataLoadPhase = "Failed"
+	DataLoadPhaseNone                 DataLoadPhase = ""
+	DataLoadPhasePending              DataLoadPhase = "Pending"
+	DataLoadPhaseLoading              DataLoadPhase = "Loading"
+	DataLoadPhaseCompleted            DataLoadPhase = "Completed"
+	DataLoadPhaseFailed               DataLoadPhase = "Failed"
+	DataLoadPhaseCompletedAndFinished DataLoadPhase = "CompletedAndFinished"
+	DataLoadPhaseFailedAndFinished    DataLoadPhase = "FailedAndFinished"
```

Comment:
I'm a little confused here. Is there a necessary to add these two phases? From my perspecitve,  the "CompletedAndFinished" and "FailedAndFinished" will confuse the users who launch the dataload.

Reply:
If not using two phases, the final state is the same while the job has two different phases “Completed" and "Failed” after it finished.

Reply:
Two phase is more clear for me.